docs/content/en/docs-dev/quickstart/_index.md

```diff
@@ -74,16 +65,24 @@ helm install piped pipecd/piped -n pipecd \
   --set secret.pipedKey.data={YOUR_PIPED_SECRET_KEY}
 ```
 
-### 5. Configuring a kubernetes application
-Navigate to the `Application` page, click on the `Add` button. Then give as:
+### 4. Registering a kubernetes application
+Navigate to the `Application` page, click on the `Add` button on the top left corner.
+
+Go to the `ADD FROM GIT` tab, then select:
+- Piped: `dev` (you just registered)
+- CloudProvider: `kubernetes-default`
+
+You should see a lot of application suggestions.
+
+Click the `canary` row. Make sure Kind is set to KUBENETES, and then click the ADD button.
```

Comment:
```suggestion
You should see a lot of suggested applications.

Select the `canary` application and click the `ADD`button to register.
```

Reply:
There are lots of apps named canary. I thought the screenshot isn't enough, that's why I added this explanation but do you think it's redundant?

Reply:
By line 73, the user already selected `kubernetes-default` provider so I guess there is only one `canary` application that will be shown, isn't that?

Reply:
You're right 👍